plugins/locale-observer.js

```diff
@@ -8,8 +8,9 @@ export default function (context) {
 }
 
 function _setLocaleCookie(locale, isDev) {
+  const localeName = new Intl.Locale(locale).baseName
```

Comment:
Utiliser la méthode `Intl.getCanonicalLocales()` dédiée pour accomplir cette action cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/getCanonicalLocales

Suggestion de code, qui sera à adapter pour prendre en compte les erreurs possibles :
```suggestion
  const canonicalLocaleName = Intl.getCanonicalLocales(locale)[0]
```

`Intl.getCanonicalLocales()` semble être présente dans tous les navigateurs à supporter tel que décrit dans https://github.com/1024pix/pix/pull/5346 → donc a priori il n'est pas nécessaire de mettre en place un polyfill pour `Intl.getCanonicalLocales()`, alors que c'est bien nécessaire pour `Intl.Locale`.

Reply:
c'est fait !